config/express.js

```diff
@@ -49,7 +50,24 @@ app.use(helmet());
 // enable CORS - Cross Origin Resource Sharing
 app.use(cors());
 
-app.use(session({ secret: config.cookieSecret, resave: true, saveUninitialized: true }));
+/* const store = new MongoDBStore({
```

Comment:
I tried this yesterday and was getting something about a circular error. Not sure if you are getting the same

Reply:
I was getting `APIError: No such user exists!` but I thought it was unrelated, and then the error went away when I commented out the mongo store.